kevm_pyk/src/kevm_pyk/gst_to_kore.py

```diff
@@ -0,0 +1,105 @@
+import json
+import logging
+from collections import OrderedDict
+from io import TextIOWrapper
+from pathlib import Path
+from typing import Final
+
+_LOGGER: Final = logging.getLogger(__name__)
+
+
+def schedule_to_kore(schedule: str) -> str:
+    return f'Lbl{schedule}' + "'Unds'EVM{}()"
+
+
+def chainid_to_kore(chainid: int) -> str:
+    return "\\dv{SortInt{}}(" + f'"{chainid}")'
+
+
+def mode_to_kore(mode: str) -> str:
+    return f'Lbl{mode}' + "{}()"
+
+
+def gst_to_kore(gst_file: Path, out_stream: TextIOWrapper, schedule: str, mode: str, chainid: int) -> None:
+
+    with open(gst_file) as data_file:
+        data = json.load(data_file, object_pairs_hook=OrderedDict)
+
+    def escape(_data):
```

Comment:
Please add typing for nested functions too. Should these begin with an `_`?

Reply:
Done.